src/cdk/schematics/ng-generate/drag-drop/index.spec.ts

```diff
@@ -0,0 +1,105 @@
+import {SchematicTestRunner} from '@angular-devkit/schematics/testing';
+import {createTestApp} from '../../testing';
+import {getFileContent} from '@schematics/angular/utility/test';
+import {Schema} from './schema';
+
+describe('CDK drag-drop schematic', () => {
+  let runner: SchematicTestRunner;
+
+  const baseOptions: Schema = {
+    name: 'foo',
+    // TODO(devversion): rename project to something that is not tied to Material. This involves
+    // updating the other tests as well because `createTestApp` is responsible for creating
+    // the project.
+    project: 'material',
+  };
+
+  beforeEach(() => {
+    runner = new SchematicTestRunner('schematics', require.resolve('../../collection.json'));
+  });
+
+  it('should create drag-drop files and add them to module', () => {
+    const tree = runner.runSchematic('drag-drop', baseOptions, createTestApp(runner));
+    const moduleContent = getFileContent(tree, '/projects/material/src/app/app.module.ts');
+    const files = tree.files;
+
+    expect(files).toContain('/projects/material/src/app/foo/foo.component.css');
+    expect(files).toContain('/projects/material/src/app/foo/foo.component.html');
+    expect(files).toContain('/projects/material/src/app/foo/foo.component.spec.ts');
+    expect(files).toContain('/projects/material/src/app/foo/foo.component.ts');
+
+    expect(moduleContent).toMatch(/import.*Foo.*from '.\/foo\/foo.component'/);
+    expect(moduleContent).toMatch(/declarations:\s*\[[^\]]+?,\r?\n\s+FooComponent\r?\n/m);
+  });
+
+  it('should add drag-drop module', () => {
+    const tree = runner.runSchematic('drag-drop', baseOptions, createTestApp(runner));
+    const moduleContent = getFileContent(tree, '/projects/material/src/app/app.module.ts');
+
+    expect(moduleContent).toContain('DragDropModule');
+  });
+
+  describe('styleext option', () => {
+    it('should respect the option value', () => {
+      const tree = runner.runSchematic(
+          'drag-drop', {styleext: 'scss', ...baseOptions}, createTestApp(runner));
+
+      expect(tree.files).toContain('/projects/material/src/app/foo/foo.component.scss');
+    });
+
+    it('should fallback to the @schematics/angular:component option value', () => {
```

Comment:
fallback -> fall back. Same goes for some of the other tests.

Reply:
Yeah, we basically have this in other test files as well. I will fix this one here, but have a follow-up for the other spec files.